File: works/python-practice/files/imc.py
weight = float(input("Digite seu peso: "))
height = float(input("Digite sua altura: "))

result = float(weight / (height * height))

if result <= 18.5:
  print("Sou Abaixo do PESO.")
elif result >= 18.5 and result <= 24.9:
  print("Peso saudável")
elif result >= 25 and result <= 30:
  print("Sobrepeso")
elif result >= 30 and result <= 39.9:
  print("Obeso")
else:
  print("Obeso Mórbido")

File: works/Algoritmo/imc.cpp
#include <iostream>
#include <fstream>
#include <iomanip>

using namespace std;

/*
7) Elabore um programa em C que receba a altura e o peso de uma pessoa e calcule o seu Índice de Massa 
Corporal (IMC). O IMC é calculado através da fórmula: IMC = peso / (altura * altura). Em seguida, exiba a 
classificação de acordo com a tabela abaixo:
 • Abaixo do peso: IMC < 18.5
 • Peso normal: 18.5 <= IMC < 24.9
 • Sobrepeso: 25 <= IMC < 29.9
 • Obesidade: IMC >= 30

*/

int main(){

    double weight;
    double height;
    
    int some;
    
    cout << "Digite sua altura: ";
    cin >> height;
    cout << "Digite seu peso: ";
    cin >> weight;
    
    cout << std::setprecision(4) << (weight / height) / height;
    
    return 0;
}